src/prefect/tasks/databricks/models.py

```diff
@@ -209,3 +209,28 @@ class AccessControlRequestForGroup(BaseModel):
 
 class AccessControlRequest(BaseModel):
     __root__: Union[AccessControlRequestForUser, AccessControlRequestForGroup]
+
+class GitSource(BaseModel):
```

Comment:
can we include extra, like [here](https://github.com/PrefectHQ/prefect/pull/5949/files), in case Databricks updates API

```suggestion
class GitSource(BaseModel, extra=Extra.allow):
```